src/main/scala/com.snowplowanalytics.iglu/ctl/Command.scala

```diff
@@ -93,32 +98,70 @@ object Command {
   val skipWarnings = Opts.flag("skip-warnings", "Don't output messages with log level less than ERROR").orFalse
   val skipChecks = Opts.option[List[Linter]]("skip-checks", s"Lint without specified linters, given comma separated\n$lintersListText").withDefault(List.empty)
 
+  // server keygen options
+  val vendorPrefix = Opts.option[String]("vendor-prefix", "Vendor prefix to associate with generated key")
+    .mapValidated(s => Server.VendorPrefix.fromString(s).toValidatedNel).withDefault(Server.VendorPrefix.Wildcard)
+
+  // database options
+  val dbPort = Opts.option[Int]("port", "Database port").withDefault(5439)
+  val dbHost = Opts.option[String]("host", "Database host address").orNone
+  val dbName = Opts.option[String]("dbname", "Database name").orNone
+  val dbUserName = Opts.option[String]("username", "Database username").orNone
+  val dbPassword = Opts.option[String]("password", "Database password").orNone
```

Comment:
@aldemirenes could you please mention env vars that will be used in case options are not passed?

Reply:
Sorry, I could not understand exactly what you mean.

Reply:
```scala
val dbPassword = Opts.option[String]("password", s"Database password ($PgPasswordEnvVar environment variable will be used if option is not present)").orNone
```